e2e-tests/src/tests/scheduler.test.ts

```diff
@@ -0,0 +1,171 @@
+import { expect, test } from '@playwright/test';
+import req from '../utilities/requests';
+import time from '../utilities/time';
+
+test.describe('Scheduling', () => {
+  const rd = Math.random() * 100;
+  const planStartTimestamp = "2021-001T00:00:00.000";
```

Comment:
:small_red_triangle_down: There is a mix of camel and snake case throughout this file. Is this expect formatting/style in a JS/TS file? I'll look it up but wanted to check our expectations generally.

Reply:
I think the two timestamp were the only occurrence of the camel case. I have changed it to snake case. @camargo, any opinion on snake vs camel ? I see you mostly use snake while @mattdailis and @dyst5422 use snake. Should we harmonize at some point ?

Reply:
I generally use camel because it's fewer characters. That's as far as my justification goes tho :) I think consistency is more important than which one we pick.